actors/badge/src/main/java/org/sunbird/badge/model/ContentBadgeAssociation.java

```diff
@@ -0,0 +1,81 @@
+package org.sunbird.badge.model;
+
+import java.io.Serializable;
+
+public class ContentBadgeAssociation implements Serializable {
```

Comment:
Need to use this class in Dao service as input/output - currently this is not used, i believe